.editorConfig

```diff
@@ -0,0 +1,30 @@
+# This file is for unifying the coding style for different editors and IDEs
```

Comment:
Please be sure that we get the file name correct this time.

`.editorConfig` -> `.editorconfig`

Also, seems we are having excess unused config options and the Ruby and Makefile stuff could perhaps be trimmed.

Reply:
OK. Done.